### ISSUE TITLE
Ability to choose what settings and add-ons should be copied for the Windows log on screen and other protected screens

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4997]
**Reported by k_kolev1985 on 2015-03-25 11:06**
At present, when we activate the button in NVDA's General Settings dialog to copy the current user's configuration to the Windows log on screen and other protected screens, NVDA alerts us that copying of add-ons may affect the security, but after we confirm, it copies them anyway (witch it should do if we confirm it). Personally I don't need most (if not all) add-ons to be loaded in the log on screen and other protected screens and I don't want them copied for use there. But at present, NVDA doesn't allow us to choose what should be copied: only the settings file; the add-ons as well; the profiles; maybe only the add-ons without the settings file; and if we choose to copy the add-ons as well, to allow us to choose witch of the add-ons should be copied and witch not (optional). To overcome this limitation, I do the following: I copy my personal settings file to the "systemConfig" folder in NVDA's installation folder. Ideally however, this should be possible through NVDA itself. What do you think?

**Comment by leonardder** [comment]
Closing this as duplicate of newer issue #6305, which contains more discussion.
